lib/config.go

```diff
@@ -475,6 +475,36 @@ func (c *RuntimeConfig) Validate(onExecution bool) error {
 		if _, err := sysregistriesv2.GetRegistries(nil); err != nil {
 			return fmt.Errorf("invalid /etc/containers/registries.conf: %q", err)
 		}
+
+		for _, hooksDir := range c.HooksDir {
+			if _, err := os.Stat(hooksDir); err != nil {
+				return errors.Wrapf(err, "invalid hooks_dir entry")
+			}
+		}
+
+		if _, err := os.Stat(c.Conmon); err != nil {
+			return errors.Wrapf(err, "invalid conmon path")
+		}
+	}
+
+	return nil
+}
+
+// Validate is the main entry point for network configuration validation.
+// The parameter `onExecution` specifies if the validation should include
+// execution checks. It returns an `error` on validation failure, otherwise
+// `nil`.
+func (c *NetworkConfig) Validate(onExecution bool) error {
+	if onExecution {
+		if _, err := os.Stat(c.NetworkDir); err != nil {
+			return errors.Wrapf(err, "unavailable network_dir")
```

Comment:
Can we make the messages consistent with the errors above? I prefer "invalid" as "unavailable" sounds like a service or process.

Apologies for missing that earlier.

Reply:
Sure, I changed it.